include/opendht/network_engine.h

```diff
@@ -399,7 +399,7 @@ class NetworkEngine final
     static const constexpr size_t NODE4_INFO_BUF_LEN {26};
     /* the length of a node info buffer in ipv6 format */
     static const constexpr size_t NODE6_INFO_BUF_LEN {38};
-    /* the maximum time we wait for an UDP reply */
+    /* the maximum time we wait for an UDP reply before telling the link layer */
```

Comment:
something like
/* after a UDP reply, the period during which we tell the link layer about it */